src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -506,11 +510,17 @@ private void processCloudWatchAttemptLogInformation(Map<String, Set<String>> com
                                                         String fileName,
                                                         CloudWatchAttemptLogFileInformation
                                                                 cloudWatchAttemptLogFileInformation) {
-        File file = new File(fileName);
+        LogFile file = new LogFile(fileName);
         String componentName = attemptLogInformation.getComponentName();
+        boolean isActiveFile = false;
+        if (ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG) {
```

Comment:
ADD A BIG TODO EXPLAINING THAT THIS IS JUST A TEMPORARY THING TO AVOID CHANGING a bunch of tests and additional code